spec/pluralrules.html

```diff
@@ -357,66 +368,78 @@ <h1>
       <h1>
         PluralRuleSelect (
           _locale_: a String,
-          _type_: a String,
+          _type_: *"cardinal"* or *"ordinal"*,
           _n_: a finite Number,
           _operands_: a Plural Rules Operands Record derived from formatting _n_,
-        )
+        ): *"zero"*, *"one"*, *"two"*, *"few"*, *"many"*, or *"other"*
       </h1>
       <dl class="header">
         <dt>description</dt>
-        <dd>It returns the String from &laquo; *"zero"*, *"one"*, *"two"*, *"few"*, *"many"*, *"other"* &raquo; that best categorizes the _operands_ representation of _n_ according to the rules for _locale_ and _type_.</dd>
+        <dd>The returned String best categorizes the _operands_ representation of _n_ according to the rules for _locale_ and _type_.</dd>
       </dl>
     </emu-clause>
 
-    <emu-clause id="sec-resolveplural" aoid="ResolvePlural">
-      <h1>ResolvePlural ( _pluralRules_, _n_ )</h1>
-      <p>
-        When the ResolvePlural abstract operation is called with arguments _pluralRules_ (which must be an object initialized as a PluralRules) and _n_ (which must be a Number value), it returns a Record with two string-valued fields describing _n_ according to the effective locale and the options of _pluralRules_: [[PluralCategory]] characterizing its <emu-xref href="#sec-pluralruleselect">plural category</emu-xref>, and [[FormattedString]] containing its formatted representation. The following steps are taken:
-      </p>
-
+    <emu-clause id="sec-resolveplural" type="abstract operation">
+      <h1>
+        ResolvePlural (
+          _pluralRules_: an Intl.PluralRules,
+          _n_: a Number,
+        ): a Record with fields [[PluralCategory]] (*"zero"*, *"one"*, *"two"*, *"few"*, *"many"*, or *"other"*) and [[FormattedString]] (a String)
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>The returned Record contains two string-valued fields describing _n_ according to the effective locale and the options of _pluralRules_: [[PluralCategory]] characterizing its <emu-xref href="#sec-pluralruleselect">plural category</emu-xref>, and [[FormattedString]] containing its formatted representation.</dd>
+      </dl>
       <emu-alg>
-        1. Assert: Type(_pluralRules_) is Object.
-        1. Assert: _pluralRules_ has an [[InitializedPluralRules]] internal slot.
-        1. Assert: Type(_n_) is Number.
         1. If _n_ is not a finite Number, then
-          1. Return *"other"*.
```

Comment:
More an editorial issue, because the current spec can't be implemented, because [ResolvePluralRange](https://tc39.es/ecma402/#sec-resolvepluralrange) accesses `[[FormattedString]]` in step 8, which is invalid when `ResolvePlural` returns a String value.